tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js

```diff
@@ -56,6 +56,22 @@ context('Multiple users. Assign task, job.', () => {
                 secondUser.emailAddr,
                 secondUser.password,
             );
+            cy.get('#cvat-create-task-button').click({ force: true });
+            cy.url().should('include', '/tasks/create');
+            cy.get('[id="name"]').type(taskName);
+            cy.get('.cvat-constructor-viewer-new-item').click();
+            cy.get('[placeholder="Label name"]').type(labelName);
+            cy.get('.cvat-new-attribute-button').click();
+            cy.get('[placeholder="Name"]').type(attrName);
+            cy.get('.cvat-attribute-type-input').click();
+            cy.get('.cvat-attribute-type-input-text').click();
+            cy.get('[placeholder="Default value"]').type(textDefaultValue);
+            cy.contains('button', 'Done').click();
```

Comment:
Looks like cypress command: ``createLabel`` or something like this.

Reply:
Fixed.